lib/rubocop/cop/rails/key_word_args.rb

```diff
@@ -0,0 +1,71 @@
+# encoding: utf-8
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop is used to identify usages of `http methods
+      # like get,post,put,path` without the usage of keyword arguments
+      # in your tests and change them to use keyword arguments.
+      #
+      # @example
+      #   # bad
+      #   get :new, { user_id: 1}
+      #
+      #   # good
+      #   get :new, params: { user_id: 1 }
+      class KeyWordArgs < Cop
+        MSG = 'Use `keyword arguments` instead of ' \
+              'normal arguments for http call: '.freeze
+        KEY_WORD_ARGS = [:headers, :env, :params, :body, :flash, :as].freeze
+        HTTP_METHODS = [:get, :post, :put, :patch, :delete, :head].freeze
+        def on_send(node)
```

Comment:
Add a blank line before the method name. I wonder why didn't generate a RuboCop offence.